src/Widgets/PowerModeList.vala

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ * SPDX-FileCopyrightText: 2024 elementary, Inc. (https://elementary.io)
+ */
+
+using Power.Services.DBusInterfaces;
```

Comment:
We don't use `using` statements in elementary code style

Reply:
Fixed